src/framework/global/thirdparty/kors_async/async/internal/channelimpl.h

```diff
@@ -112,7 +112,7 @@ class ChannelImpl : public Asyncable::IConnectable
                 }
 
                 if (r) {
-                    assert(mode != Asyncable::Mode::SetOnce && "callback is already setted");
+                    //assert(mode != Asyncable::Mode::SetOnce && "callback is already set");
```

Comment:
Where this assertion occurs, simply specify `async::Asyncable::Mode::SetReplace` as the third argument in `onReceive`  Like this https://github.com/musescore/MuseScore/pull/30763 

We are trying to eliminate or clearly mark places where re-subscription occurs.

Reply:
Thanks a lot!!

Reply:
Done